source/user/adc.rst

```diff
@@ -1,97 +1,180 @@
-.. _theory_adc:
+.. _user_adc:
 
 ************************************************
 Algebraic diagrammatic construction (ADC) scheme
 ************************************************
 
 *Modules*: :mod:`adc`
 
-Background: Propagator Theory
+Introduction
 =============================
+The algebraic diagrammatic construction theory (ADC(n)) is a post-Hartree-Fock
+method used for computing correlated excited states of molecules.
+:cite:`Schirmer1983,Schirmer1998`
+The ADC methods involve a perturbative expansion of a propagator followed
+by truncation at a certain order :math:`n` that defines the ADC(n) approximation.
+Depending upon the property being investigated, propagators can be of different
+types, which lead to different variants of the ADC method.
```

Comment:
OK, but you should still specify which propagators are implemented in the adc module PySCF and which aren't :)